component/main.jsonnet

```diff
@@ -129,9 +129,62 @@ local organizationNamespaces = kyverno.ClusterPolicy('organization-namespaces')
   },
 };
 
+/**
+  * ClusterRole allowed to create and edit reserved namespaces
+  */
+local editReservedNamespacesClusterRole = kube.ClusterRole(params.reservedNamespaces.allowedClusterRoles.name) {
+  rules: [],
+};
+
+/**
+  * Refute create or edit reserved namespaces
```

Comment:
I would avoid putting too much into 1 single `main.jsonnet` file since it could grow very large over time. Maybe add its own file like `namespace-policies.jsonnet`?

Reply:
👍